R/outputs_heatmap.R

```diff
@@ -194,8 +194,7 @@
 
     if (length(x[[.heatMapColData]])) {
         # Reordering by the column annotations.
-        additional <- c(additional, sprintf(".column_annot_order <- with(.column_data, order(%s))",
-            paste(x[[.heatMapColData]], collapse=", ")))
+        additional <- c(additional, sprintf(".column_annot_order <- order(%s)", paste(sprintf('.column_data[["%s"]]', x[[.heatMapColData]]), collapse=", ")))
```

Comment:
Should be `deparse`ing the individual elements of `x[[.heatMapColData]]` instead:

```r
sprintf(".column_data[[%s]]", vapply(x[[.heatMapColData]], deparse, ""))
```